pages/design/rule-design.md

```diff
@@ -39,34 +79,64 @@ A test aspect is a part of a test subject that must be available in order to pro
 
-## Applicability
+For more details, see [ACT Rules Format: Assumptions](https://www.w3.org/TR/act-rules-format/#assumptions).
 
-Applicability describes which (elements of) web pages should be tested using the rule. These elements are known as test targets. Applicability must be written in plain language, so that it can be used by QA testers. Applicability must rely on well defined properties of the technologies that are tested. For example, a rule may be applicable to all `video` elements, but it can not be applicable to all `object` elements used to show video, unless the term "video" is further defined.
+## Accessibility Support
 
-Finding objective definitions to use in rules can be difficult, if not outright impossible in some cases. The intent here is to ensure repeatability of the rule. Not everything in WCAG testing is entirely repeatable, but when it comes to rule applicability, this is a hard requirement.
+Sometimes a feature relevant to a rule is not fully supported by assistive technology and user agents. Include information on any known support issues for a feature relevant to the rule.
 
-For more details, see [ACT Rules Format: Applicability](https://w3c.github.io/wcag-act/act-rules-format.html#applicability)
+For more details, see [ACT Rules Format: Accessibility Support](https://www.w3.org/TR/act-rules-format/#accessibility-support).
 
-## Expectations
+## Background
+
+While optional, this provides background information relevant to the development of the rule. Any document cited in the rule should be included here, along with any applicable success criterion, understanding or technique documents, other specifications, and any other relevant reading. 
+
+For more details, see [ACT Rules Format: Background](https://www.w3.org/TR/act-rules-format/#background).
+
+## Test cases
+
+The test cases are snippets of code that help with understanding and can be used for validating implementations of the rule. There must be at least one example for **pass**, **fail** and **inapplicable** outcomes. A passing or inapplicable example should pass other applicable rules. A failing example should only fail in the one aspect being demonstrated.
 
-The applicability help the testers (or test tools) identify what has to be checked. Following that, the expectations are statements that must be true for the applicable elements to pass the rule. If any of the expectations is false, than the target element failed the rule.
+Each test case should be simply and uniquely named, _e.g. "Passed Example 1"_. Each example must also include a brief declarative description that uses plain language, and is a well-formed grammatically correct sentence that finishes with a full stop/period. Words from code should use back ticks and be identified as elements or attributes.
```

Comment:
```suggestion
Each test case should be simply and uniquely named, _e.g. "Passed Example 1"_. Each must also include a brief description that explains why the example has the outcome it claims to have. 

The description should:
- use declarative plain language in the format: _This [test target] [meets / does not meet condition] [because optional reasons]._ 
- fully qualify the relevant test target, such as a particular element, or element with id, or first such element, or absent element.
- state that the example _has_ or _does not have_ or _contains_ the necessary condition to be met, and optionally any clarifying reasons. 
- use sentences that are well-formed, grammatically correct and finish with a full stop/period. 
- use back ticks around any words from code and be identify them as elements or attributes.
```

Reply:
@WilcoFiers @Jym77 does the above suggested change meet your previous comments and what was agreed in #834 ?

I've also suggested a change to the example, and am wondering if a couple of examples would be more helpful.

Reply:
Seems good.
* On the second item "fully qualify…", I think the enumeration is not closed and should end with "…" rather than full period.
* (pedantic typographic mode) if you end your items with full period, the next one should start with an uppercase letter. These items should probably end with a semi-colon instead (except the last one) (and I usually try to do "bla bla**; and**" to emphasise the link between the items (conjunction vs disjonction)).

Reply:
@Jym77 I've updated the suggestion. Is that what you meant?

Reply:
I've also updated it. It is OK for you?

Reply:
@Jym77 amended ... typical English phraseology would only include "and" before the last item of a list.

I think lack of other comments can consider this resolved.

Reply:
I'm OK with that.
Hammerring in the "and" is not really useful here.

Minor nerdy concern that this list is semi-colon separated and not full sentences while the list in the "rule name" section is full sentences and period separated (and both look somewhat similar). But I'm OK with the current situation.

Reply:
I did consider that and decided it was OK to handle them differently. The "rule name" stuff is more complex to explain and much more specific in its requirements.